<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StudentForm</class>
 <widget class="QMainWindow" name="StudentForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>437</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Student</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <item>
       <widget class="QLabel" name="lblName">
        <property name="text">
         <string>Name: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblStudentID">
        <property name="text">
         <string>Student ID: </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblGender">
        <property name="text">
         <string>Gender: </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <property name="topMargin">
       <number>8</number>
      </property>
      <property name="bottomMargin">
       <number>8</number>
      </property>
      <item>
       <widget class="QLabel" name="lblFaculty">
        <property name="text">
         <string>Faculty: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblMajor">
        <property name="text">
         <string>Major: </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblClass">
        <property name="text">
         <string>Class: </string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>594</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>Grade Table</string>
     </property>
     <addaction name="acLineSeries"/>
     <addaction name="acBarSeries"/>
    </widget>
    <addaction name="acQuerySingleSubject"/>
    <addaction name="menu_2"/>
    <addaction name="acExport"/>
    <addaction name="separator"/>
    <addaction name="acExit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="acQuerySingleSubject">
   <property name="text">
    <string>Query Single Grade</string>
   </property>
  </action>
  <action name="acExport">
   <property name="text">
    <string>Export</string>
   </property>
  </action>
  <action name="acExit">
   <property name="text">
    <string>Logout</string>
   </property>
   <property name="toolTip">
    <string>Logout</string>
   </property>
  </action>
  <action name="acLineSeries">
   <property name="text">
    <string>Line Graph</string>
   </property>
  </action>
  <action name="acBarSeries">
   <property name="text">
    <string>Bar Graph</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
